docs/detections/prebuilt-rules/tune-rule-signals.asciidoc

```diff
@@ -139,19 +132,19 @@ organizations. Different networks conform to different security policies,
 standards, and regulations. When normal network activity triggers alerts,
 network rules can be disabled or modified. For example:
 
-* To exclude a specific source, add a `not source.ip` statement with the
-relevant IP address, and a `destination.port` statement with the relevant port
-number (`not source.ip : 196.1.0.12 and destination.port : 445`).
-* To exclude source network traffic for an entire subnet, add a `not source.ip`
-statement with the relevant CIDR notation (`not source.ip : 192.168.0.0/16`).
+* To exclude a specific source, add a `source.ip` exception with the
+relevant IP address, and a `destination.port` exception with the relevant port
+number (`source.ip is 196.1.0.12` and `destination.port is 445`).
+* To exclude source network traffic for an entire subnet, add a `source.ip`
+exception with the relevant CIDR notation (`source.ip is 192.168.0.0/16`).
```

Comment:
@FrankHassanabad - is using CIDR notation in exception OK?

Reply:
Yes the plumbing should all work there for it.